tests/test_integrations.py

```diff
@@ -0,0 +1,9 @@
+"""Test integrations that work with unittest testcases."""
+# pylint: disable=missing-docstring,no-self-use
+import unittest
+
+from tests.features import FlexmockTestCase
+
+
+class TestIntegrations(FlexmockTestCase, unittest.TestCase):  # pylint: disable=too-many-ancestors
```

Comment:
```suggestion
class TestGenericIntegration(FlexmockTestCase, unittest.TestCase):  # pylint: disable=too-many-ancestors
```
we could also rename the file similarly. I think it carries the intent better

Reply:
https://github.com/flexmock/flexmock/pull/114/commits/393dec2123b45f98243a0e2e5953af16c9072f37